connect/reports/renderers/base.py

```diff
@@ -34,9 +43,48 @@ def get_context(self, data):
     def set_extra_context(self, data):
         self.extra_context = data
 
+    def render(self, data, output_file, start_time=NOW):
+        with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
You should check if start_time is None and if so, set it as now using pytz to have it as utc.

Reply:
Implemented